examples/DotNetCore/WebDemo/Microsoft.Azure.AppConfiguration.WebDemo/appsettings.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "Logging": {
+    "IncludeScopes": false,
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
+
+  "Settings": {
+    "AppName": "Azure App Configuration Demo",
+    "Version": 1.0,
+    "FontSize": 50,
+    "RefreshRate": 1000,
+    "Language": "English",
+    "Messages": "Hello There;Thanks For Using Azure App Configuration",
+    "BackgroundColor": "Orange"
+  },
+
+  "AzureAppConfigurationEndpoint": "Replace with App Config Endpoint",
```

Comment:
>Replace with App Config Endpoint [](start = 36, length = 32)

Don't give this a value or else the program will see that this is not null or empty and then will try to use this as the endpoint. Make it "" and then write a comment.

Reply:
Replaced it with an empty string.